javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -823,7 +844,11 @@ void PrintProtoDtsMessage(Printer* printer, const Descriptor* desc,
   printer->Indent();
   for (int i = 0; i < desc->field_count(); i++) {
     const FieldDescriptor* field = desc->field(i);
-    vars["js_field_name"] = JSFieldName(field);
+
+    string base_js_field_name = JSFieldName(field);
+    string accessor_name = SafeAccessorName(base_js_field_name);
+
+    vars["js_field_name"] = accessor_name;
```

Comment:
nit: maybe just simplify like this? :)

vars["js_field_name"] = SafeAccessorName(JSFieldName(field));